src/Lean/Server/References.lean

```diff
@@ -72,30 +72,39 @@ def merge (a : RefInfo) (b : RefInfo) : RefInfo :=
     usages := a.usages.append b.usages
   }
 
-def contains (self : RefInfo) (pos : Lsp.Position) : Bool := Id.run do
+def findRange? (self : RefInfo) (pos : Lsp.Position) (includeStop := false) : Option Range := do
```

Comment:
```suggestion
def findRange? (self : RefInfo) (pos : Lsp.Position) (includeEnd := false) : Option Range := do
```
since the field is `range.end`

Reply:
It's called `stop` in lean's own data structures like `Std.Range.stop`. The only reason why `Lsp.Range` uses `end` is because the LSP spec says so